package.json

```diff
@@ -8,7 +8,8 @@
     "check:release": "sui-studio check-release",
```

Comment:
this shouldn't be here, right?

Reply:
What do you mean? I can't see any differences from the file in master